messages/sk.json

```
@@ -0,0 +1,131 @@
+  "omnishipping.shippingEstimate-m": "{timeAmount, plural, =0 {práve teraz} one {o # minútu} other {do # minút}}",
+  "omnishipping.shippingEstimate-db": "Pracovné dni",
+  "omnishipping.free": "Zadarmo",
+  "omnishipping.shippingEstimatePickup-bd": "{timeAmount, plural, =0 {Pripravené dnes} one {Pripravené za # pracovný deň} other {Pripravené za # pracovné dni (pracovných dní)}}",
+  "omnishipping.shippingEstimatePickup-d": "{timeAmount, plural, =0 {Pripravené dnes} one {Pripravené za # deň} other {Pripravené za # dni (dní)}}",
+  "omnishipping.shippingEstimatePickup-h": "{timeAmount, plural, =0 {Pripravené na vyzdvihnutie} one {Pripravené za # hodinu} other {Pripravené za # hodiny (hodín)}}",
+  "omnishipping.shippingEstimatePickup-m": "{timeAmount, plural, =0 {Pripravené na vyzdvihnutie} one {Pripravené za # minútu} other {Pripravené za # minúty (minút)}}",
+  "omnishipping.scheduled": "Naplánované",
+  "omnishipping.scheduledDeliveryChoice": "Vyberte dátum dodania",
+  "omnishipping.scheduledDeliverySplitDisclaimer": "Niektoré položky nie je možné naplánovať. Vyberte spôsob doručenia pre tieto položky.",
+  "omnishipping.scheduledDeliveryPartial": "{amount, plural, =0 { } one {Naplánovaná # položka} other {Naplánované # položky (Naplánovaných # položiek}}\n​",
+  "omnishipping.scheduledDeliveryPrice": "Ceny od {price}",
+  "omnishipping.scheduledDeliveryAll": "Naplánovať všetko",
+  "omnishipping.deliveryOptionsLeft": "Spôsob doručenia zostávajúcich položiek",
+  "omnishipping.selectPickupPoint": "Vybrať odberné miesto na Vyzdvihnutie v obchode",
+  "omnishipping.selectDeliveryOptions": "Vyberte možnosti doručenia",
+  "omnishipping.pointDetails": "Podrobnosti o odbernom mieste na Vyzdvihnutie v obchode",
+  "omnishipping.closeButton": "zavrieť",
+  "omnishipping.deliveryDate": "Plánovanie",
+  "omnishipping.deliveryTime": "Dátum doručenia",
+  "omnishipping.datepickerDateFormat": "dddd, LL",
+  "omnishipping.deliveryWindows": "Od {startDate} do {endDate}",
+  "omnishipping.deliveryWindowsSummary": "{date}, medzi {startDate} a {endDate}",
+  "omnishipping.deliveryWindowsSummaryDate": "{date}, medzi {startDate} a {endDate}",
+  "omnishipping.deliveryWindowsSummaryTime": "medzi {startDate} a {endDate}",
+  "omnishipping.list": "Zoznam",
+  "omnishipping.map": "Mapa",
+  "omnishipping.pickupPoint.modify": "Zmeniť",
+  "omnishipping.modify": "upraviť",
+  "omnishipping.modifyPostalCode": "Zmeniť",
+  "omnishipping.details": "Viac informácií",
+  "omnishipping.change": "Zobraziť všetky dostupné odberné miesta na Vyzdvihnutie v obchode",
+  "omnishipping.searchLocationMap": "Vyhľadajte ulicu, PSČ, štvrť alebo mesto",
+  "omnishipping.productsInPoint": "Výrobky sú dostupné v tomto odbernom mieste",
+  "omnishipping.aditionalInfo": "Ďalšie informácie",
+  "omnishipping.unavailable": "nedostupné",
+  "omnishipping.unavailableItemsAmount": "{itemsAmount, plural, =0 {} one {# položka nedostupná} other {# položky nedostupné (# položiek nedostupných}}",
```

I think this should be
```suggestion
  "omnishipping.unavailableItemsAmount": "{itemsAmount, plural, =0 {} one {# položka nedostupná} other {# položky nedostupné {# položiek nedostupných}}",
```

---

changing this would cause a syntax error I think, because there would be a missing `}` in the end.. this message seems to be weird

---

@dvm1984 What should be the message in this case?

---

Looking at similar messages + syntax + translation, I now think it should be `)` on the other side
```suggestion
  "omnishipping.unavailableItemsAmount": "{itemsAmount, plural, =0 {} one {# položka nedostupná} other {# položky nedostupné (# položiek nedostupných)}",
```